<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationWindow</class>
 <widget class="QMainWindow" name="RegistrationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>952</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="nutzertyp_page">
         <widget class="QGroupBox" name="groupBox">
          <property name="geometry">
           <rect>
            <x>340</x>
            <y>170</y>
            <width>211</width>
            <height>141</height>
           </rect>
          </property>
          <property name="title">
           <string>Ich möchte die Applikation nutzen als:...</string>
          </property>
          <widget class="QRadioButton" name="radioButton">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>30</y>
             <width>82</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Arzt</string>
           </property>
          </widget>
          <widget class="QRadioButton" name="radioButton_2">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>70</y>
             <width>82</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Patient</string>
           </property>
          </widget>
          <widget class="QRadioButton" name="radioButton_3">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>110</y>
             <width>82</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Angehöriger</string>
           </property>
          </widget>
         </widget>
         <widget class="QPushButton" name="next_btn_1">
          <property name="geometry">
           <rect>
            <x>754</x>
            <y>512</y>
            <width>161</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>weiter</string>
          </property>
         </widget>
         <widget class="QPushButton" name="back_btn_1">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>520</y>
            <width>161</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>zurück</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_2">
         <widget class="QPushButton" name="next_btn_2">
          <property name="geometry">
           <rect>
            <x>750</x>
            <y>510</y>
            <width>161</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>weiter</string>
          </property>
         </widget>
         <widget class="QPushButton" name="back_btn_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>510</y>
            <width>161</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>zurück</string>
          </property>
         </widget>
         <widget class="QWidget" name="gridLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>350</x>
            <y>100</y>
            <width>331</width>
            <height>211</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="1">
            <widget class="QLineEdit" name="user_le">
             <property name="placeholderText">
              <string>Benutzername</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="password2_le">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
             <property name="placeholderText">
              <string>********</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="password_le">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
             <property name="placeholderText">
              <string>********</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Benutzername:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Passwort:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Passwort bestätigen:</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>952</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
